### ISSUE TITLE
Upgrading alfresco CE installed through installer

### ISSUE THREAD

**Issue by shodr** [Alfresco/alfresco-docker-installer#62]
Hi, 

This is not an issue, but could not find the proper procedure to upgrade alfresco and dependant components installed through alfresco-docker-installer.

Before I go through messing with .env and .yml file, and troubleshooting, has anybody tried upgrading. If so, can they share changes they made and specify to which version of acs and other components they upgraded, and procedure (will DB work as is or it has to be reimported, etc.) 

Tnx

**Comment by aborroy** [comment]
I guess the easy approach is to generate the Docker Compose resources for the new version and restore all your data, configuration and addons from your previous version.

There are different upgrading procedures in the official documentation, so you can identify what needs to be preserved.

**Comment by shodr** [comment]
Hi, i tried looking through the documentation before, but could not find upgrade path.  In docker repository I can see versions 6.2.2-XY and 7. for core app, newer share versions,  transformers etc. Tried comparing with AC deployment yml for community edition, it uses version 7 for repo, but there is no info regarding upgrading etc.

Because of bug in ACA, documents are not displayed immediately, I played around with ACA master-latest (1.8) and upgraded it to master. Now documents are immediately displayed, but preview is not working, I guess some transformer dependencies. 

If there exists documentation I would greatly appreciate link to it.
